components/brave_vpn/BUILD.gn

```diff
@@ -10,9 +10,8 @@ import("//tools/grit/preprocess_if_expr.gni")
 preprocess_folder = "preprocessed"
 preprocess_mojo_manifest = "preprocessed_mojo_manifest.json"
 
-assert(enable_brave_vpn)
-
 static_library("brave_vpn") {
+  assert(enable_brave_vpn)
```

Comment:
why this change?

Reply:
`enabled_brave_vpn` is false in certain configurations (linux at the moment), but this build file is used for JS-only compiles as of this PR. So I moved the assert from the build-file-level to the browser c++ build level.